data/mods/Xedra_Evolved/recipes/dreamforged_weapon.json

```diff
@@ -74,7 +74,7 @@
     "components": [ [ [ "forged_dreamstuff_ingot", 5 ] ] ],
     "difficulty": 4,
     "flags": [ "SECRET" ],
-    "time": "21 h"
+    "time": "7 h"
```

Comment:
Did you decide the ones after this didn't need proficiencies?

Reply:
Profs are inherited by `"copy-from": "dreamforged_longsword_ench",`